slim/Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN set -x \
     && cd / && rm -rf /znc-src
 
 COPY docker-entrypoint.sh /
-
+RUN chmod +x /docker-entrypoint.sh
```

Comment:
When i've tried build the image for arm , i've got some errors relative to this, saying that i dont have permissions to run the script. After i've added this line , the error was fixed .